setup.py

```diff
@@ -59,7 +59,12 @@
 elif '--use_vitisai' in sys.argv:
     package_name = 'onnxruntime-vitisai'
     sys.argv.remove('--use_vitisai')
-# --use_acl is specified in build.py, but not parsed here
+elif '--use_acl' in sys.argv:
```

Comment:
I built onnxruntime(and python whl) with your forked master, passing --use_acl but the onnxruntime whl had armnn in its name. Also, the installed package is called onnxruntime-armnn:
```
onnxruntime@onnxruntime-desktop:~/Documents/tmp/onnxruntime/build/Linux/Release/dist$ ls *.whl 
onnxruntime_armnn-1.3.1-cp36-cp36m-linux_aarch64.whl
onnxruntime@onnxruntime-desktop:~/Documents/tmp/onnxruntime/build/Linux/Release/dist$ pip3 freeze | grep onnx
onnxruntime-armnn==1.3.1
onnxruntime@onnxruntime-desktop:~/Documents/tmp/onnxruntime/build/Linux/Release/dist$ python3
Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
>>> onnxruntime.get_available_providers()
['CPUExecutionProvider', 'ACLExecutionProvider']
>>> 
```

Reply:
tools/ci_build/build.py wrong argument order in build_python_wheel for wheel_name_suffix